docs/GraphQL/makingASimpleRequest.rst

```diff
@@ -1,4 +1,88 @@
 Making a simple request
 ============================
 
-TODO
\ No newline at end of file
+This guide is intended to give an example on how to make your first query and mutation on the platform. We will use the provided playground on `api.passi.is/playground <https://api.passi.is/playground>`_ to try our the requests to the graphQL server.
+
+We will assume that you have already have made your first pass template and wish to add new passes to it. We will also use the cookie provided from the UI to make the test request but there will be an example on how to use the api key in code examples. To generate a cookie to use, simply log on to `passi.is <https://passi.is>`_.
+
+What we first want to do is get the Id of the passtemplate we want to create the new pass with. To do this we will query the passTemplates to get a list of the passtemplates we have. 
+
+Because the list is paged, the data we are looking for will be returned under data. We only really need to find the Id here but because we might need something to distinguish between the templates if we have multiple, so we fetch the name as well.
+
+.. code-block::
+
+    query passTemplateQuery {
+        passTemplates {
+            data {
+                id
+                name
+            }
+        }
+    }
+
+
+*Note that passtemplates accept a option of specifying how many results we get per page, but we will leave it at the default for this example.*
+
+When you have written the query into the playground you can hit the play button to execute the query. This should give us a reply which looks something like this.
+
+.. code-block:: json
+
+    {
+        "data": {
+            "passTemplates": {
+                "data": [
+                    {
+                        "id": "d9d42dfa-6d58-4b2a-f173-85ec9100f077",
+                        "name": "VIP club membership"
+                    },
+                    {
+                        "id": "247851ef-88a6-4004-9312-1e0a2c3b94de",
+                        "name": "no descirption test hehe"
+                    }
+                ]
+            }
+        }
+    }
+
+From there we can grab the ID we need and start creating passes.
+
+To create a pass we have to use the upsertPass mutation and pass in the id as a parameter. To do this we write the following query.
```

Comment:
Maybe use another word for passing in a value? 
Since we are often talking about passes it might be confusing.
A suggestion: 
To create a pass we have to use the upsertPass mutation and send in the id as a parameter. To do this we write the following query.

Reply:
I completely agree.